integration/tracing/tracing-opentelemetry/src/main/java/org/apache/cxf/tracing/opentelemetry/OpenTelemetryFeature.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.tracing.opentelemetry;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Tracer;
+import org.apache.cxf.Bus;
+import org.apache.cxf.annotations.Provider;
+import org.apache.cxf.annotations.Provider.Scope;
+import org.apache.cxf.annotations.Provider.Type;
+import org.apache.cxf.common.injection.NoJSR250Annotations;
+import org.apache.cxf.feature.AbstractPortableFeature;
+import org.apache.cxf.feature.DelegatingFeature;
+import org.apache.cxf.interceptor.InterceptorProvider;
+
+@NoJSR250Annotations
+@Provider(value = Type.Feature, scope = Scope.Server)
+public class OpenTelemetryFeature extends DelegatingFeature<OpenTelemetryFeature.Portable> {
+    public OpenTelemetryFeature() {
+        super(new Portable());
+    }
+
+    public OpenTelemetryFeature(final Tracer tracer) {
+        super(new Portable(tracer));
+    }
+
+    public static class Portable implements AbstractPortableFeature {
+        private OpenTelemetryStartInterceptor in;
+        private OpenTelemetryStopInterceptor out;
+
+        public Portable() {
+            this(GlobalOpenTelemetry.getTracer("CXFServer"));
```

Comment:
For "CXFServer", we probably need to extract the constant like DEFAULT_TRACER_NAME

Reply:
Thank you, shared my thoughts here https://github.com/apache/cxf/pull/1340#discussion_r1276905543